web/src/components/navbar/NavBarDesktop.tsx

```diff
@@ -26,12 +26,19 @@ export const NavBarDesktop = (): ReactElement => {
   const anchorRef = useRef<HTMLButtonElement | null>(null);
 
   const toggleDropdown = (): void => {
+    if (currentlyOnboarding()) {
+      return;
+    }
     if (!open) {
       analytics.event.account_name.click.expand_account_menu();
     }
     setOpen((prevOpen) => !prevOpen);
   };
 
+  const currentlyOnboarding = (): boolean => {
+    return router.pathname === "/onboarding" ? true : false;
```

Comment:
this ternary can be simplified, the true/false is redundant, you can just say `return router.pathname === "/onboarding"`